### PR TITLE
UI/UX Polish

### LINKED ISSUE #54: Final Sweep: UX
- [x] make sure all buttons look good and draw intiuitively draw attention 
- [x] check the z index of all elements on the page
- [x] impliment fonts
- [x] impliment smaller animations

### PR DESCRIPTION
**Pull request by BrigetteDoelp**
## Description
Polishes up the UI of the application. 
## Motivation and Context
We wanted the application to have an even look across all styling application wide.
## How Has This Been Tested?
- [ ] In console
- [x] In browser
- [ ] With testing files
- [ ] Other Please specify:
## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Styling / Design
- [ ] Testing
- [x] Refactor
## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
#### Issues Addressed:
Resolves #54

### CODE REVIEW

File: src/Components/App/App.scss [hunk 1]
```diff
@@ -1,20 +1,13 @@
...
-//   height: 3vh;
-//   border: solid 1px #121316;
-//   background-color: #121316;
-//   width: 20vw;
-//   margin: 1%;
-// }
-
-:focus{
-  outline: none;
+  font-family: 'Abel', sans-serif;
```
Comment (nathanielmillard) [inline src/Components/App/App.scss:4]: Glad to see we got these fonts implimented, do really feel like it makes a difference.

File: public/index.html [hunk 2]
```diff
@@ -9,6 +9,7 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link href="https://fonts.googleapis.com/css2?family=Abel&family=Dosis:wght@500;700;800&family=Spartan:wght@300;500;700;900&display=swap" rel="stylesheet">
```
Comment (Kristiannmiller) [inline public/index.html:12]: loving the new fonts! Looks great!

File: src/Components/App/App.js [hunk 3]
```diff
@@ -46,9 +46,12 @@ class App extends Component {
           <Route path='/myreflections'>
             <ReflectionLibrary />
           </Route>
-          <Route path='/breathing/:feeling'>
-            <BreathingRoom/>
-          </Route>
+          <Route path='/breathing/:feeling' render={({match}) => {
```
Comment (Kristiannmiller) [inline src/Components/App/App.js:49]: This is SO helpful! Glad we had the chance to address this issue with the page reloading and leaving our code lost without a mood prop.

File: src/variables.scss [hunk 4]
```diff
@@ -0,0 +1,43 @@
+$lightest: hsla(200, 14%, 96%, 1);
```
Comment (nathanielmillard) [inline src/variables.scss:1]: Glad to see this file added for future simplification of color naming!

File: src/Components/BreathingRoom/BreathingRoomAnimation.js [hunk 5]
```diff
@@ -1,59 +1,69 @@
...
 
-export const inhaleExhale = () => {
+export const inhaleExhale = (mood) => {
+  let colors = {
+    fine: ['hsl(128, 29%, 90%)', 'hsl(200, 14%, 96%)', 'hsl(119, 38%, 72%)',
+      'hsl(120, 18%, 56%)', 'hsl(124, 11%, 24%)', 'hsl(0, 0%, 8%)'],
+    anxious: ['hsl(49, 43%, 71%)', 'hsl(200, 14%, 96%)', 'hsl(48, 46%, 51%)',
+      'hsl(48, 57%, 45%)', 'hsl(48, 44%, 25%)', 'hsl(225, 10%, 8%)'],
+    angry: ['hsl(325, 15%, 78%)', 'hsl(200, 14%, 96%)', 'hsl(327, 18%, 52%)',
+        'hsl(329, 57%, 25%)', '	hsl(327, 52%, 21%)', 'hsl(320, 30%, 12%)'],
```
Comment (Kristiannmiller) [inline src/Components/BreathingRoom/BreathingRoomAnimation.js:10]: There appears to be a giant space in one of these strings. Is this a GitHub problem or a spacing issue we need to address?

**Review by nathanielmillard — APPROVED** [review]
Looking good! glad to be on our finishing sweeps for polishing up the smaller details.

File: src/Components/FeelingRoom/fineAnimations.js [hunk 6]
```diff
@@ -13,23 +13,23 @@ export const fineBackgroundAnimation = () => {
 }
 
 export const floatAnimation = (element, scale) => {
-  let floatTl = gsap.timeline({repeat: 200, });
+  let floatTl = gsap.timeline({repeat: 200, yoyo:true});
```
Comment (Kristiannmiller) [inline src/Components/FeelingRoom/fineAnimations.js:16]: This is a great solution for getting the bubbles to gather again without being so jarring. Nice work

File: src/Components/BreathingRoom/BreathingRoomAnimation.js (hunk 5 above)
Comment (nathanielmillard) [reply src/Components/BreathingRoom/BreathingRoomAnimation.js:10]: I'll take a look on my syntax sweep later today :)

File: src/Components/ReflectionLibrary/ReflectionLibrary.js [hunk 7]
```diff
@@ -34,7 +34,7 @@ class ReflectionLibrary extends Component {
   render() {
     return (
       <section data-testid='reflections'>
-      {console.log(this.state.reflections)}
+        { this.state.errorMessage && <h1>{this.state.errorMessage}</h1>}
```
Comment (Kristiannmiller) [inline src/Components/ReflectionLibrary/ReflectionLibrary.js:37]: Great error handling here!

File: src/variables.scss (hunk 4 above)
Comment (Kristiannmiller) [inline src/variables.scss:1]: This is great! I know we have a lot of styled components, and using these might be tricky, but I'm excited to implement them down the road and it's nice to have them "ready to go"

**Review by Kristiannmiller — APPROVED** [review]
Love all these little style tweaks and additions! It's all really looking great! Let's get some testing done on the next branch

**Pull request merged**